cypress/jenkins/cypress.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -e
+
+pwd
+ls -al
+cd "dashboard"
+
+yarn install
+yarn add --dev -W mocha mochawesome mochawesome-merge \
```

Comment:
comment only. how come the switch to mocha was needed?

Reply:
It seems that Cypress makes use of mocha libraries as a backend for a few things like reporting. I needed to include these  during execution to generate jUnit and html reports and merge them for Jenkins publihsing.

Reply:
cypress should dump out some reasonable html reports, think probably junit as well. though i don't think that should block this PR, and you can tailor the output without having to worry about breaking github workflows